### ISSUE TITLE
«The product that was requested doesn't exist. Verify the product and try again.» at  vendor/magento/module-catalog/Model/ProductRepository.php:310

### ISSUE THREAD

**Issue by dmitry-fedyuk** [tradefurniturecompany/site#20]

**Comment by dmitry-fedyuk** [comment]
```
{
    "productId": "0",
    "storeId": "1"
}
Array
(
    [0] => Array
        (
            [File] => vendor/magento/module-catalog/Model/ProductRepository.php
            [Line] => 316
            [Caller] => Magento\Catalog\Model\ProductRepository::getById
            [Callee] => df_log_l
        )

    [1] => Array
        (
            [File] => vendor/magento/module-sales/CustomerData/LastOrderedItems.php
            [Line] => 135
            [Caller] => Magento\Sales\CustomerData\LastOrderedItems::getItems
            [Callee] => Magento\Catalog\Model\ProductRepository::getById
        )

    [2] => Array
        (
            [File] => vendor/magento/module-sales/CustomerData/LastOrderedItems.php
            [Line] => 195
            [Caller] => Magento\Sales\CustomerData\LastOrderedItems::getSectionData
            [Callee] => Magento\Sales\CustomerData\LastOrderedItems::getItems
        )

    [3] => Array
        (
            [File] => vendor/magento/module-customer/CustomerData/SectionPool.php
            [Line] => 93
            [Caller] => Magento\Customer\CustomerData\SectionPool::getAllSectionData
            [Callee] => Magento\Sales\CustomerData\LastOrderedItems::getSectionData
        )

    [4] => Array
        (
            [File] => vendor/magento/module-customer/CustomerData/SectionPool.php
            [Line] => 60
            [Caller] => Magento\Customer\CustomerData\SectionPool::getSectionsData
            [Callee] => Magento\Customer\CustomerData\SectionPool::getAllSectionData
        )

    [5] => Array
        (
            [File] => vendor/magento/module-customer/Controller/Section/Load.php
            [Line] => 78
            [Caller] => Magento\Customer\Controller\Section\Load::execute
            [Callee] => Magento\Customer\CustomerData\SectionPool::getSectionsData
        )

    [6] => Array
        (
            [File] => vendor/magento/framework/Interception/Interceptor.php
            [Line] => 58
            [Caller] => Magento\Customer\Controller\Section\Load\Interceptor::___callParent
            [Callee] => Magento\Customer\Controller\Section\Load::execute
        )

    [7] => Array
        (
            [File] => vendor/magento/framework/Interception/Interceptor.php
            [Line] => 138
            [Caller] => Magento\Customer\Controller\Section\Load\Interceptor::Magento\Framework\Interception\{closure}
            [Callee] => Magento\Customer\Controller\Section\Load\Interceptor::___callParent
        )

    [8] => Array
        (
            [File] => vendor/magento/framework/Interception/Interceptor.php
            [Line] => 153
            [Caller] => Magento\Customer\Controller\Section\Load\Interceptor::___callPlugins
            [Callee] => Magento\Customer\Controller\Section\Load\Interceptor::Magento\Framework\Interception\{closure}
        )

    [9] => Array
        (
            [File] => generated/code/Magento/Customer/Controller/Section/Load/Interceptor.php
            [Line] => 26
            [Caller] => Magento\Customer\Controller\Section\Load\Interceptor::execute
            [Callee] => Magento\Customer\Controller\Section\Load\Interceptor::___callPlugins
        )

    [10] => Array
        (
            [File] => vendor/magento/framework/App/Action/Action.php
            [Line] => 108
            [Caller] => Magento\Framework\App\Action\Action::dispatch
            [Callee] => Magento\Customer\Controller\Section\Load\Interceptor::execute
        )

    [11] => Array
        (
            [File] => vendor/magento/framework/Interception/Interceptor.php
            [Line] => 58
            [Caller] => Magento\Customer\Controller\Section\Load\Interceptor::___callParent
            [Callee] => Magento\Framework\App\Action\Action::dispatch
        )

    [12] => Array
        (
            [File] => vendor/magento/framework/Interception/Interceptor.php
            [Line] => 138
            [Caller] => Magento\Customer\Controller\Section\Load\Interceptor::Magento\Framework\Interception\{closure}
            [Callee] => Magento\Customer\Controller\Section\Load\Interceptor::___callParent
        )

    [13] => Array
        (
            [File] => vendor/magento/framework/Interception/Interceptor.php
            [Line] => 153
            [Caller] => Magento\Customer\Controller\Section\Load\Interceptor::___callPlugins
            [Callee] => Magento\Customer\Controller\Section\Load\Interceptor::Magento\Framework\Interception\{closure}
        )

    [14] => Array
        (
            [File] => generated/code/Magento/Customer/Controller/Section/Load/Interceptor.php
            [Line] => 39
            [Caller] => Magento\Customer\Controller\Section\Load\Interceptor::dispatch
            [Callee] => Magento\Customer\Controller\Section\Load\Interceptor::___callPlugins
        )

    [15] => Array
        (
            [File] => vendor/magento/framework/App/FrontController.php
            [Line] => 159
            [Caller] => Magento\Framework\App\FrontController::processRequest
            [Callee] => Magento\Customer\Controller\Section\Load\Interceptor::dispatch
        )

    [16] => Array
        (
            [File] => vendor/magento/framework/App/FrontController.php
            [Line] => 99
            [Caller] => Magento\Framework\App\FrontController::dispatch
            [Callee] => Magento\Framework\App\FrontController::processRequest
        )

    [17] => Array
        (
            [File] => vendor/magento/framework/Interception/Interceptor.php
            [Line] => 58
            [Caller] => Magento\Framework\App\FrontController\Interceptor::___callParent
            [Callee] => Magento\Framework\App\FrontController::dispatch
        )

    [18] => Array
        (
            [File] => vendor/magento/framework/Interception/Interceptor.php
            [Line] => 138
            [Caller] => Magento\Framework\App\FrontController\Interceptor::Magento\Framework\Interception\{closure}
            [Callee] => Magento\Framework\App\FrontController\Interceptor::___callParent
        )

    [19] => Array
        (
            [File] => vendor/magento/module-store/App/FrontController/Plugin/RequestPreprocessor.php
            [Line] => 94
            [Caller] => Magento\Store\App\FrontController\Plugin\RequestPreprocessor::aroundDispatch
            [Callee] => Magento\Framework\App\FrontController\Interceptor::Magento\Framework\Interception\{closure}
        )

    [20] => Array
        (
            [File] => vendor/magento/framework/Interception/Interceptor.php
            [Line] => 135
            [Caller] => Magento\Framework\App\FrontController\Interceptor::Magento\Framework\Interception\{closure}
            [Callee] => Magento\Store\App\FrontController\Plugin\RequestPreprocessor::aroundDispatch
        )

    [21] => Array
        (
            [File] => vendor/magento/module-page-cache/Model/App/FrontController/BuiltinPlugin.php
            [Line] => 73
            [Caller] => Magento\PageCache\Model\App\FrontController\BuiltinPlugin::aroundDispatch
            [Callee] => Magento\Framework\App\FrontController\Interceptor::Magento\Framework\Interception\{closure}
        )

    [22] => Array
        (
            [File] => vendor/magento/framework/Interception/Interceptor.php
            [Line] => 135
            [Caller] => Magento\Framework\App\FrontController\Interceptor::Magento\Framework\Interception\{closure}
            [Callee] => Magento\PageCache\Model\App\FrontController\BuiltinPlugin::aroundDispatch
        )

    [23] => Array
        (
            [File] => vendor/magento/framework/Interception/Interceptor.php
            [Line] => 153
            [Caller] => Magento\Framework\App\FrontController\Interceptor::___callPlugins
            [Callee] => Magento\Framework\App\FrontController\Interceptor::Magento\Framework\Interception\{closure}
        )

    [24] => Array
        (
            [File] => generated/code/Magento/Framework/App/FrontController/Interceptor.php
            [Line] => 26
            [Caller] => Magento\Framework\App\FrontController\Interceptor::dispatch
            [Callee] => Magento\Framework\App\FrontController\Interceptor::___callPlugins
        )

    [25] => Array
        (
            [File] => vendor/magento/framework/App/Http.php
            [Line] => 137
            [Caller] => Magento\Framework\App\Http::launch
            [Callee] => Magento\Framework\App\FrontController\Interceptor::dispatch
        )

    [26] => Array
        (
            [File] => vendor/magento/framework/Interception/Interceptor.php
            [Line] => 58
            [Caller] => Magento\Framework\App\Http\Interceptor::___callParent
            [Callee] => Magento\Framework\App\Http::launch
        )

    [27] => Array
        (
            [File] => vendor/magento/framework/Interception/Interceptor.php
            [Line] => 138
            [Caller] => Magento\Framework\App\Http\Interceptor::Magento\Framework\Interception\{closure}
            [Callee] => Magento\Framework\App\Http\Interceptor::___callParent
        )

    [28] => Array
        (
            [File] => vendor/magento/framework/Interception/Interceptor.php
            [Line] => 153
            [Caller] => Magento\Framework\App\Http\Interceptor::___callPlugins
            [Callee] => Magento\Framework\App\Http\Interceptor::Magento\Framework\Interception\{closure}
        )

    [29] => Array
        (
            [File] => generated/code/Magento/Framework/App/Http/Interceptor.php
            [Line] => 26
            [Caller] => Magento\Framework\App\Http\Interceptor::launch
            [Callee] => Magento\Framework\App\Http\Interceptor::___callPlugins
        )

    [30] => Array
        (
            [File] => vendor/magento/framework/App/Bootstrap.php
            [Line] => 261
            [Caller] => Magento\Framework\App\Bootstrap::run
            [Callee] => Magento\Framework\App\Http\Interceptor::launch
        )

    [31] => Array
        (
            [File] => pub/index.php
            [Line] => 40
            [Caller] => 
            [Callee] => Magento\Framework\App\Bootstrap::run
        )

)

```

**Comment by dmitry-fedyuk** [comment]
[tradefurniturecompany.co.uk/admin/sales/order/view/order_id/34688](https://www.tradefurniturecompany.co.uk/admin/sales/order/view/order_id/34688)

![02](https://user-images.githubusercontent.com/83204/74792471-c688d280-52e7-11ea-8ae9-89b4720c7015.png)

[tradefurniturecompany.co.uk/admin/catalog/product/edit/id/84](https://www.tradefurniturecompany.co.uk/admin/catalog/product/edit/id/84)

![01](https://user-images.githubusercontent.com/83204/74792374-7578de80-52e7-11ea-96c4-133704a12923.png)

**Comment by dmitry-fedyuk** [comment]
Solved in [tradefurniturecompany/core 0.0.6](https://github.com/tradefurniturecompany/core/releases/tag/0.0.6).
